### ISSUE TITLE
Cache

### ISSUE THREAD

**Issue by silsha** [connectordb/connectordb-chrome#4]
To reduce network requests we should cache the activity for x and transfer them in a single request.

**Comment by dkumor** [comment]
Yep! This was in the pipeline for several months now, but the chrome extension has not gotten much attention (most of the work has been going on in the frontend, getting plotting and visualization running).

Originally, the idea was to put a caching class directly in connectordb.js, similarly to the Logger in python. But, it is probably best to just use `chrome.storage` for now.

**Comment by dkumor** [comment]
The current version has the necessary structure implemented, but caching itself is not implemented. It would take more time than I have available this week (need to finish the frontend plotting and release b1!) to do this correctly, so I am unassigning myself for now.
